ts/Dashboards/Components/EditableOptions.ts

```diff
@@ -95,7 +96,20 @@ class EditableOptions {
     }
 
     public getOptions(): (Array<EditableOptions.Configuration>) {
-        return this.component.options.editableOptions;
+        const options = this.component.options.editableOptions;
+        for (let i = 0; i < options.length; i++) {
```

Comment:
Please assign length into variable, it will increase performance.

Eg: `for (let i = 0, iEnd = options.length; i < iEnd; ++i) {`